package.json

```diff
@@ -21,10 +22,12 @@
     "query-string": "^4.3.1",
     "react": "16.0.0-alpha.6",
     "react-native": "0.44.2",
+    "react-native-actionsheet": "^2.2.0",
     "react-native-communications": "^2.2.1",
     "react-native-device-info": "^0.10.2",
     "react-native-elements": "^0.9.3",
     "react-native-htmlview": "^0.9.0",
+    "react-native-material-design-searchbar": "^1.1.4",
```

Comment:
Yep we've discussed this in #2. So you'll notice that I already use `react-native-search-bar` instead of the RNE solution. That library provides a wrapper around a native iOS search bar instead of relying on a non-native solution.

We've discussed that it probably makes sense to use a separate library for Android and keep the iOS logic as native as possible. However I have two questions:

1. Is there any sort of actual native Android search bar with a react-native bridge we can use instead?
2. If 1 doesn't seem to exist - then that's okay. The last time I tried using the search bar with RNE - it wasn't the simplest to modify and use. I don't know if that's changed however. I think it's worth a quick check and if we think that using this imported library than I'm okay with that